src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+
+    Animal.alive.push(this);
+  }
+
+  death() {
+    Animal.alive.splice(Animal.alive.indexOf(this), 1);
```

Comment:
Avoid nesting loops. Use filter instead of splice(indexOf)

Reply:
@andrii-dmytruk, Thank you, dude